spark-job-server-proposal.md

```diff
@@ -0,0 +1,61 @@
+# Adding Spark Job Server To Run Spark Jobs
+
+## Introduction
+
+This document provides design of spark job server which would trigger spark
+jobs either as scheduled or  batch jobs. Providing the user a ui for inputing
```

Comment:
Are we talking about always scheduling the spark apps as Kubernetes jobs?  If so, we could use CronJobs for scheduling.